### PR TITLE
Clean up tabbing and warnings in mempool feature.

### PR DESCRIPTION
**Pull request by mikedennis**
In prep for refactoring.

### CODE REVIEW

File: Stratis.Bitcoin/Features/MemoryPool/MempoolOrphans.cs [hunk 1]
```diff
@@ -14,327 +14,327 @@ namespace Stratis.Bitcoin.Features.MemoryPool
...
-		    foreach (var txin in tx.Inputs)
-		    {
-			    // TODO: this goes in the RelayBehaviour
-			    //CInv _inv(MSG_TX | nFetchFlags, txin.prevout.hash);
-			    //behavior.AttachedNode.Behaviors.Find<RelayBehaviour>() pfrom->AddInventoryKnown(_inv);
-			    //if (!await this.AlreadyHave(txin.PrevOut.Hash))
-			    //	from. pfrom->AskFor(_inv);
-		    }
-		    var ret = await this.AddOrphanTx(from.PeerVersion.Nonce, tx);
+            //int nFetchFlags = 0; //GetFetchFlags(pfrom, chainActive.Tip(), chainparams.GetConsensus());
```
Comment (bokobza) [inline Stratis.Bitcoin/Features/MemoryPool/MempoolOrphans.cs]: is this not used anywhere?

File: Stratis.Bitcoin/Features/MemoryPool/MempoolValidator.cs [hunk 2]
```diff
@@ -26,889 +26,891 @@ public interface IMempoolValidator
...
+        private readonly TxMempool memPool;
+        private readonly PowConsensusValidator consensusValidator;
+        private readonly ILogger logger;
+        public MempoolPerformanceCounter PerformanceCounter { get; }
+        public PowConsensusOptions ConsensusOptions => this.consensusValidator.ConsensusOptions;
+        public static readonly FeeRate MinRelayTxFee = new FeeRate(DefaultMinRelayTxFee);
+        // TODO: Implement Later with CheckRateLimit() 
+        //private readonly FreeLimiterSection freeLimiter;
+
+        //private class FreeLimiterSection
```
Comment (bokobza) [inline Stratis.Bitcoin/Features/MemoryPool/MempoolValidator.cs]: is this not used anywhere either?

File: Stratis.Bitcoin/Features/MemoryPool/MempoolValidator.cs [hunk 3]
```diff
@@ -26,889 +26,891 @@ public interface IMempoolValidator
...
+        private readonly ConcurrentChain chain;
+        private readonly CoinView coinView;
+        private readonly TxMempool memPool;
+        private readonly PowConsensusValidator consensusValidator;
+        private readonly ILogger logger;
+        public MempoolPerformanceCounter PerformanceCounter { get; }
+        public PowConsensusOptions ConsensusOptions => this.consensusValidator.ConsensusOptions;
+        public static readonly FeeRate MinRelayTxFee = new FeeRate(DefaultMinRelayTxFee);
+        // TODO: Implement Later with CheckRateLimit() 
+        //private readonly FreeLimiterSection freeLimiter;
```
Comment (bokobza) [inline Stratis.Bitcoin/Features/MemoryPool/MempoolValidator.cs]: is this not used?

File: Stratis.Bitcoin/Features/MemoryPool/MempoolValidator.cs [hunk 4]
```diff
@@ -26,889 +26,891 @@ public interface IMempoolValidator
...
+                Money nDeltaFees = context.ModifiedFees - context.ConflictingFees;
+                if (nDeltaFees < MinRelayTxFee.GetFee(context.EntrySize))
+                {
+                    context.State.Fail(MempoolErrors.Insufficientfee,
+                            $"rejecting replacement {context.TransactionHash}, not enough additional fees to relay; {nDeltaFees} < {MinRelayTxFee.GetFee(context.EntrySize)}").Throw();
+                }
+            }
+        }
+
+        private void Chec
```
Comment (bokobza) [inline Stratis.Bitcoin/Features/MemoryPool/MempoolValidator.cs]: why did this get commented?

**Comment by mikedennis** [comment]
Yes the only changes were commenting out some code that wasn't used anywhere that was throwing up a couple of compiler warnings. It compiles clean now.

**Comment by bokobza** [comment]
Lovely stuff, thank you :-)
Can you please delete the line with nFetchFlags? Then I'll merge it.

**Comment by bokobza** [comment]
you can remove it completely, the GetFetchFlags doesn't even exist.

**Comment by bokobza** [comment]
Thank you Mike :-) @mikedennis

**Pull request merged**